GUI/QtQuick/QtQuickAdapter.cpp

```diff
@@ -2378,3 +2382,13 @@ void QtQuickAdapter::notifyNewTransaction(const bs::TXEntry& tx)
       txDetails->disconnect(this);
    }, Qt::QueuedConnection);
 }
+
+void QtQuickAdapter::exportTransaction(const QUrl& path)
```

Comment:
I think we agreed to pass `QTXSignRequest*` as an arg here, otherwise, how to obtain it?